Kernel/kprintf.cpp

```diff
@@ -191,3 +196,4 @@ extern "C" void kernelearlyputstr(char const* characters, size_t length)
     for (size_t i = 0; i < length; ++i)
         internal_dbgputch(characters[i]);
 }
+#endif
```

Comment:
Essentially, all of these syscall file changes should be their own commit. more or less.